matchms/filtering/clean_compound_name.py

```diff
@@ -42,6 +42,14 @@ def remove_known_non_compound_parts(name):
             name = name.replace(part, "")
         return name.strip("; ")
 
+    def remove_misplaced_mass(name):
+        """Remove occationally occuring parent mass addition to name."""
+        regex_mass = r"^[0-9]{2,3}.[0-9]$"
```

Comment:
This regex seems to come from experience from spectra with names with masses. The `.` will match any char. You probably want to match an actual dot char.
```suggestion
        regex_mass = r"^[0-9]{2,4}\.[0-9]$"
```

Any reason to have a max of `999.9`?

Reply:
Thanks, good catch with the point. And I now extended it to 2-4 digits which will also allow masses > 999.9.